sopel/modules/remind.py

```diff
@@ -391,7 +391,7 @@ def get_duration(self, today=None):
 
         .. seealso::
 
-            The :mod:`datetime` built-in module's documentation explains what
+            The :mod:`datetime` built-in plugin's documentation explains what
```

Comment:
NAK: This is referring to a stdlib module.

(Global find-and-replace is kinda hard, huh? 🤣)

Reply:
LOL.